scenarios/netops/mqttToKafkaBatchingBridge/converter.py

```diff
@@ -0,0 +1,116 @@
+import paho.mqtt.client as mqtt
+import json
+import sys
+from datetime import datetime, timedelta
+import time 
+from cloud_publisher import KafkaPublisher
+import logging
+import json
+
+
+
+LOG_FORMAT_STRING = "%Y-%m-%d"
+LOG_LEVEL = "INFO"
+
+def get_formatted_datetime():
+    now = datetime.now()     
+    return now.strftime(LOG_FORMAT_STRING)
+
+
+logPath = "./log"
+fileName = f"mqtt_logs_{get_formatted_datetime()}"
+
+logFormatter = logging.Formatter("%(asctime)s [%(threadName)-12.12s] [%(levelname)-5.5s]  %(message)s")
+rootLogger = logging.getLogger("mini-batcher-application")
+rootLogger.setLevel(logging.DEBUG)
+
+fileHandler = logging.FileHandler("{0}/{1}.log".format(logPath, fileName))
+fileHandler.setFormatter(logFormatter)
+fileHandler.setLevel(logging.DEBUG)
+rootLogger.addHandler(fileHandler)
+
+consoleHandler = logging.StreamHandler()
+consoleHandler.setFormatter(logFormatter)
+consoleHandler.setLevel(logging.DEBUG)
+rootLogger.addHandler(consoleHandler)
+
+args = sys.argv
+
+mqtt_host = args[1] 
```

Comment:
Actually, since it is dockerized, using os environment variables is an even better choice. It should take a couple of mins to add this feature. I will add it now :)